packages/core/src/text/Label.tsx

```diff
@@ -1,9 +1,11 @@
 import { forwardRef } from "react";
 import { Text, TextProps } from "./Text";
 
+export type LabelProps = Omit<TextProps<"label">, "as">;
+
 export const Label = forwardRef<
   HTMLLabelElement,
-  Omit<TextProps<"label">, "as">
+  LabelProps
```

Comment:
Is this necessary? I would avoid mixing this in

Reply:
makes sense, done